packages/react-saasify/src/components/NavHeader/NavHeader.js

```diff
@@ -88,27 +88,27 @@ export class NavHeader extends Component {
                 <div className={theme(styles, 'actions')}>
                   <Link to='/logout' className={theme(styles, 'login')}>
                     <CTAButton type='secondary' inline>
-                      LOGOUT
+                      Log out
```

Comment:
agreed; this should be handled via css.

Reply:
Yup - I think the existing themes already use uppercase.

Worth noting I also turned it into a verb/action ("Log out") rather than a noun ("Logout").